pfcpiface/up4.go

```diff
@@ -207,65 +198,74 @@ func (up4 *UP4) setupChannel() error {
 	return nil
 }
 
-func (up4 *UP4) initAllCounters() error {
-	log.Debug("Initializing counter for UP4")
+func (up4 *UP4) initAllCounters() {
+	log.Debug("Initializing counters for UP4")
 
-	err := up4.initCounter(preQosCounterID, "PreQosPipe.pre_qos_counter")
-	if err != nil {
-		return ErrOperationFailedWithReason("init preQosCounterID counter", err.Error())
+	counters := []uint32{
+		p4constants.CounterPreQosPipePreQosCounter,
+		p4constants.CounterPostQosPipePostQosCounter,
 	}
 
-	err = up4.initCounter(postQosCounterID, "PostQosPipe.post_qos_counter")
-	if err != nil {
-		return ErrOperationFailedWithReason("init postQosCounterID counter", err.Error())
-	}
+	for _, counterID := range counters {
+		counterName := p4constants.GetCounterIDToNameMap()[counterID]
 
-	return nil
+		counterSize, err := up4.p4RtTranslator.getCounterSizeByID(counterID)
+		if err != nil {
+			log.Error(err)
+		}
+
+		switch counterID { //FIXME initCounter should accept a uint32 but doing so will break integration tests (PFCP Agent cannot connect). Need to investigate on why
```

Comment:
@osinstom any idea on why using uint32 instead of uint8 breaks integration tests?